addOns/evalvillain/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Changed
+- Fix the downloaded version of the Eval Villain Firefox extension (Issue 7008).
```

Comment:
The referenced issue is about Chrome not this specific issue, might worth tweaking.